README.md

```diff
@@ -7,6 +7,7 @@ This repository contains code for a webhook which monitors the creation of Servi
 The following steps will install the webhook into an existing OpenShift deployment, instructions for kubernetes will be similar
 
 * download your secret and create a file called secret.yaml
+* go get github.com/knrc/registry-puller/cmd
```

Comment:
It should only require 'go get github.com/knrc/registry-puller' as this pulls it in to the gopath, can you update and remove '/cmd' ?

Reply:
ok, done.